vnext/Desktop/React.Windows.Desktop.vcxproj

```diff
@@ -77,18 +77,14 @@
       -->
       <PreprocessorDefinitions>
         BOOST_ASIO_HAS_IOCP;
-        _WINSOCK_DEPRECATED_NO_WARNINGS;
-        _WIN32_WINNT=$(WinVer);
+        _WINSOCK_DEPRECATED_NO_WARNINGS; _WIN32_WINNT=$(WinVer);
```

Comment:
Please revert. Leave one entry per line for readability.

Reply:
this is still a draft, will clean things up before ready for real review. Azure devops doesn't allow CI to run on a separate github repo, so I have to create a dummy PR to get it to run...

Reply:
Sounds good. Unblocking.